HLTrigger/Muon/src/HLTL1MuonNoL2Selector.cc

```diff
@@ -87,38 +84,35 @@ void HLTL1MuonNoL2Selector::produce(edm::StreamID, edm::Event& iEvent, const edm
   edm::Handle<SeedMap> seedMapHandle;
   iEvent.getByToken(seedMapToken_, seedMapHandle);
   
-  std::vector<l1t::MuonRef> firedL1Muons_;
-  edm::Handle<trigger::TriggerFilterObjectWithRefs> L1Cands;
-  iEvent.getByToken(theL1CandToken_, L1Cands);
-  L1Cands->getObjects(trigger::TriggerL1Mu, firedL1Muons_);
-      
   for (int ibx = muColl->getFirstBX(); ibx <= muColl->getLastBX(); ++ibx) {
-    if (ibx != 0) continue;
+    if (centralBxOnly_ && (ibx != 0)) continue;
     for (auto it = muColl->begin(ibx); it != muColl->end(ibx); it++){
       l1t::MuonRef l1muon(muColl, distance(muColl->begin(muColl->getFirstBX()),it) );
       
-      // only select L1's that fired: 
-      if(find(firedL1Muons_.begin(), firedL1Muons_.end(), l1muon) == firedL1Muons_.end()) continue;
-      
+      unsigned int quality = it->hwQual();
+      float pt    =  it->pt();
+      float eta   =  it->eta();
+
+      if ( pt < theL1MinPt_ || std::abs(eta) > theL1MaxEta_  || quality <= theL1MinQuality_) continue;
+
```

Comment:
Do I understand correctly that the change is to apply a given quality / pT / eta cut on the L1 candidates, instead of matching them to the output of the L1 filter ?
At which step of the path do we recover the consistent HLT-L1 matching ?